Makefile.am

```diff
@@ -213,6 +213,56 @@ clean-local:
 	-rm -rf t/cover_db/
 
 t/cover_db/: check
+COVER_OPTS= \
+	PERL5OPT="-MDevel::Cover=-db,$(abs_builddir)/cover_db,+inc,$(PWD),-select,$(PWD)/[^/]+.pm|$(PWD)/(OpenQA|consoles|backend)|$(PWD)/isotovideo,-ignore,/usr/bin/prove|t/data/|t/fake/|\.t|tools/lib/perlcritic"
```

Comment:
this looks very dangerous as now we need to explicitly add new stuff to the list when we add new files or directories. Can we keep the old way or work with a blacklist instead?

Reply:
I was trying a lot but I found no other way to collect all files.
But since in this PR I still fo `cd t` I can try with the old options.
*Ideally* we would have all modules in `lib/` and all scripts in `bin/` so we wouldn't have to struggle with this, but I guess moving all the modules into lib would be a bigger task.